### PR TITLE
add review

### PR DESCRIPTION
**Pull request by baaohuii**

### CODE REVIEW

**Comment by baaohuii** [comment]
ready

File: app/assets/stylesheets/reviews.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Reviews controller here.
```
Comment (ngocvt-0484) [inline app/assets/stylesheets/reviews.scss]: plz remove all empty file you not use

File: app/controllers/reviews_controller.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+class ReviewsController < ApplicationController
+  
+  def new
+  	@review = Review.new
```
Comment (ngocvt-0484) [inline app/controllers/reviews_controller.rb]: tab 2 space please

File: app/helpers/reviews_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module ReviewsHelper
```
Comment (ngocvt-0484) [inline app/helpers/reviews_helper.rb]: remove it

File: app/models/review.rb [hunk 4]
```diff
@@ -0,0 +1,4 @@
+class Review < ApplicationRecord
+  belongs_to :users
```
Comment (ngocvt-0484) [inline app/models/review.rb]:
```suggestion
  belongs_to :user
```

File: app/models/review.rb [hunk 5]
```diff
@@ -0,0 +1,4 @@
...
+  belongs_to :courses
```
Comment (ngocvt-0484) [inline app/models/review.rb]:
```suggestion
  belongs_to :course
```

**Comment by baaohuii** [comment]
ready

File: app/controllers/reviews_controller.rb [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+  def create
+  	@review = Review.new review_params
+  	@review.user_id = current_user.id
+
+  	if @review.save
+  	  redirect_to user_path
+  	else
+  	  render :new
+  	end
+  end
```
Comment (saobv-2223) [inline app/controllers/reviews_controller.rb]: indenting ONLY 2 spaces. please

File: app/views/reviews/_review.html.erb [hunk 7]
```diff
@@ -0,0 +1 @@
+<p><%= review.comment %></p>
```
Comment (saobv-2223) [inline app/views/reviews/_review.html.erb]:
```suggestion
<p><%= review.comment %></p>

```

File: config/locales/en.yml [hunk 8]
```diff
@@ -78,3 +78,10 @@ en:
       the_form_contains: "The form contains "
       confirmation: "Confirmation"
       create_acc: "Create my account"
+    
+  reviews:
+    new:
+      add: "add review"
+      new_review: "New Review"
+      create_review: "Create review"
+
```
Comment (saobv-2223) [inline config/locales/en.yml]:
```suggestion
      create_review: "Create review"
```

**Comment by baaohuii** [comment]
ready

File: config/locales/en.yml (hunk 8 above)
Comment (chungpv-1008) [inline config/locales/en.yml]: please add endline

**Comment by baaohuii** [comment]
ready

**Pull request merged**